Modules/Numerics/Optimizersv4/include/itkConjugateGradientLineSearchOptimizerv4.hxx

```diff
@@ -101,6 +106,18 @@ ConjugateGradientLineSearchOptimizerv4Template<TInternalComputationValueType>::A
   this->InvokeEvent(IterationEvent());
 }
 
+/**
+ * Set the initial learning rate for optimization.
+ */
```

Comment:
Please, move the documentation to the header file. Also, since getter and setter method docs come and are displayed in pairs, please modify the doc for this.

Reply:
Docstrings now moved to the definition